### DISCUSSION TITLE
Console doesn't auto update

### DISCUSSION THREAD

**Discussion by pac78275** [semaphoreui/semaphore#1079] [General]
The console when running a playbook doesn't auto update with new output. I have to actually refresh the page to make it update. Is there some setting or something that I've missed to enable this or is this a bug? I've tried on Safari, Firefox, and Chrome, all to the same result. If this is indeed a bug I'll file an issue. I just wanted to see if there was something I might be missing before doing so.

**Comment by pac78275** [comment]
Notes:
- I installed via snap on Ubuntu 20.04
- I'm on v2.8.69
- My tasks are defined as generic tasks rather than builds or deploys as they don't fall into either of those categories

**Comment by pac78275** [comment]
I've noticed the console on the demo site does auto update which only serves to perplex me further

**Comment by tboerger** [comment]
We got an issue tracking exactly this bug, please subscribe to https://github.com/ansible-semaphore/semaphore/issues/1754 to receive updates related to that.
